components/BottomNavigation/README.md

```diff
@@ -181,3 +181,56 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+## Accessibility {#a11y}
+
+To help ensure your bottom navigation item is accessible to as many users as possible, please
+be sure to review the following recommendations:
+
+### `-accessibilityLabel`
```

Comment:
Where did this text come from? Shouldn't it also be in `accessibility.md` ?

Reply:
I just copied an existing components text.

Reply:
I thought README.md is automatically generated? Is it the other way around?

Reply:
I didn't run the script, I just typed it and copied what buttons has.